tests/test_powercurve_ops.py

```diff
@@ -0,0 +1,16 @@
+from unittest import TestCase
+
+from nrel.hive.resources.mock_lobster import mock_bev, mock_vehicle, mock_dcfc_charger
+from nrel.hive.model.vehicle.mechatronics.powercurve.powercurve_ops import time_to_full
+
+
+class TestPowercurveOps(TestCase):
+    def test_time_to_fill(self):
```

Comment:
give it a name that implies what is being tested, something like `test_time_to_fill_when_near_full`

Reply:
Agreed, fixed